ROADMAP.md

```diff
@@ -66,3 +43,40 @@ We use the [milestone](https://github.com/rook/rook/milestones) feature in Githu
     * Multi-site configuration [#1584](https://github.com/rook/rook/issues/1584)
   * Mgr and plugins
     * Placement group balancer support (enable the mgr module)
+* CockroachDB
+  * Secure deployment using certificates [#1809](https://github.com/rook/rook/issues/1809)
+  * Helm chart deployment [#1810](https://github.com/rook/rook/issues/1810)
+* EdgeFS
+  * Declare EdgeFS CRDs to be Beta v1 [#2506](https://github.com/rook/rook/issues/2506)
+  * Automatic host validation [#2409](https://github.com/rook/rook/issues/2409)
+  * Target VDEVs
+    * Support for flexible write I/O “sync” option [#2367](https://github.com/rook/rook/issues/2367)
+  * Object
+    * OpenStack/SWIFT CRD [#2509](https://github.com/rook/rook/issues/2509)
+    * Support for S3 bucket as DNS subdomain [#2510](https://github.com/rook/rook/issues/2510)
+  * Block (iSCSI)
+    * Support Block CSI [#2507](https://github.com/rook/rook/issues/2507)
+  * Mgr
+    * Support for Prometheus Dashboard and REST APIs [#2401](https://github.com/rook/rook/issues/2401)
+    * Support for Management GUI with automated CRD wizards [#2508](https://github.com/rook/rook/issues/2508)
+* Minio
+  * Helm chart deployment [#1814](https://github.com/rook/rook/issues/1814)
+* NFS
+  * Dynamic NFS provisioning [#2062](https://github.com/rook/rook/issues/2062)
+  * Client access control [#2283](https://github.com/rook/rook/issues/2283)
+
+## Beyond 1.0
```

Comment:
how about support for more "dynamic" clusters in the roadmap, e.g. GKE? #2107 
I think that has value and makes good architectural sense too.

Reply:
Awww, It does have value =) I ended up here after investigating the viability for me to use Rook on GKE.